static/js/composebox_typeahead.js

```diff
@@ -58,6 +58,7 @@ export function update_emoji_data() {
                 emoji_collection.push({
                     emoji_name: alias,
                     emoji_code: emoji_dict.emoji_code,
+                    reaction_type: emoji_dict.reaction_type,
```

Comment:
I think we probably want to add `reaction_type` in the `is_realm_emoji` case as well, so `reaction_type` is always present.

Reply:
added! 👍